modules/ROOT/pages/administration/access-control/load-privileges.adoc

```diff
@@ -0,0 +1,80 @@
+:description: How to use Cypher to manage load privileges.
+
+////
+[source, cypher, role=test-setup]
+----
+CREATE ROLE roleLoadAllData
+----
+////
+
+[role=enterprise-edition aura-db-enterprise]
+[[access-control-load-privileges]]
+= Load privileges
+
+[abstract]
+--
+This section explains how to use Cypher to manage load privileges.
+--
+
+All load privileges are relevant system-wide.
+Like DBMS privileges, they do not belong to one specific database or graph.
+For more details on the differences between graphs, databases and the DBMS, refer to xref::introduction/cypher_neo4j.adoc[].
+
+image::privileges_grant_and_deny_syntax_load_privileges.svg[title="Syntax of GRANT and DENY load Privileges"]
+
+// TODO: add image later when there is more than one LOAD privilege
+//image::privileges_hierarchy_load.svg[title="Load privileges hierarchy"]
+
+The load privileges applies to the xref:clauses/load-csv.adoc[`LOAD CSV` clause], filtering whether the data can be loaded from the given source.
+
+The load privileges are assignable using Cypher administrative commands.
+They can be granted, denied and revoked like other privileges.
+
+[NOTE]
+====
+More details about the syntax descriptions can be found xref:administration/index.adoc#administration-syntax[here].
+====
```

Comment:
I would prefer this note after the table. Otherwise people (like me) might follow the link before having read the basic syntax, at which point the syntax details aren't very useful

Reply:
I think it's generally before our syntax descriptions elsewhere but I can move it down 🤷